src/deployment/azuredeploy.bicep

```diff
@@ -98,6 +98,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2021-10-01' = {
   location: location
   properties: {
     enabledForDiskEncryption: false
+    enabledForDeployment: true
```

Comment:
What's this extra setting for?

Reply:
Geneva agent install + update process needs to have this set on KeyVault to use the certificate on the agent.